### ISSUE TITLE
Not enough hugs.

### ISSUE THREAD

**Issue by kamisori** [Ybalrid/FreeHugsBSD#1]
Needs more hugs.
*hug*

**Comment by rodizio1** [comment]
No hugs o give here (or I didn't find the right button), so you get a heart ;)

**Comment by KizzyCode** [comment]
`*hug*`

**Comment by akuckartz** [comment]
:huhu:

**Comment by akuckartz** [comment]
:hug:

**Comment by cajuncoder** [comment]
*hug*

**Comment by cavac** [comment]
*hug*

**Comment by h1z1** [comment]
Damn beat me to it.  Had one created too!   In all seriousness though - :hugs:

Still can't believe this even happened.  Maybe a requirement of GSOC this year?

**Comment by Ybalrid** [comment]
**hugs** :smile:

When I'll get some time, I'm going to look into the kernel code to see if we can modify `kern.ostype` to the value `FreeHugsBSD`.

But I have no idea what I'm doing... xD

The only thing I know is that `/sys` is the source code of the kernel, (the rest of the repository seems to be user-space programs)

**Comment by Ybalrid** [comment]
It's now done, the code in this repository *is* an os called FreeHugsBSD

**Comment by kamisori** [comment]
Perhaps a *hug* should flash on the screen at random intervals for 100ms? (perhaps as a bootoption)

It would definetly set this apart from those unhugers.

**Comment by Ybalrid** [comment]
@kamisori Well, we definitively need a logo for the boot-screen, and maybe just for the project itself.

**Comment by Ybalrid** [comment]
@cavac To be honest, I don't think that emoji looks that good. I find it hard to associate it with a hug. It's just a head with hands. you need *arms* to give hugs ^^"

**Comment by lineber** [comment]
![image](https://user-images.githubusercontent.com/2454244/36626513-1fbcb474-1902-11e8-89db-a0b1eda8bb68.png)

**Comment by Ybalrid** [comment]
@lineber you made this? If so, can you paste this a text in a code block (you put 3 " ` " characters at the beginning and end of the bloc) and add some devil's horns to this guy? :wink:

**Comment by kamisori** [comment]
No offense, but that looks scary.

**Comment by Ybalrid** [comment]
@kamisori Well, maybe it's part of the joke? xD

I don't really know. If somebody can make something that looks not scary, I'm all for it. It just has to be fun ;-)

ASCII-art is required for solving issue #5, but a logo/mascot as an image can be useful too. If somebody has some artistic talents...

**Comment by rleigh-codelibre** [comment]
`:hug:`

**Comment by lineber** [comment]
Ybalrid, no I found it on a website that had Ascii-Art. I think they did have the same thing with horns too. lol

**Comment by kamisori** [comment]
And done:

![hugs](https://user-images.githubusercontent.com/293332/36689556-03ce630e-1b30-11e8-8e9b-5760c586ee9a.jpg)

**Comment by Ybalrid** [comment]
@kamisori OH. This is absolutely awesome!

**Comment by Ybalrid** [comment]
No, really, this is supper cute, little hearts on the dress and everything

**Comment by kamisori** [comment]
My wife says, the only conditions on using her work in FreeHugsBSD, is that she is mentioned as Cipni and that no SJW harrass her about how she could promote such vile things as free hugs and such.

**Comment by Ybalrid** [comment]
This is absolutely fine with me. A mention `Logo/Mascot drew by Cipni for the FreeHugsBSD project` is fine?

**Comment by kamisori** [comment]
Would a mention of her deviantart be possible? cipni.deviantart.com

**Comment by KizzyCode** [comment]
Nice mascot!
You don't have this by any chance as alpha-layered PNG?

**Comment by kamisori** [comment]
Sadly no, my wifes computer gave up on her at some point while drawing it. The project file was lost, she finished it using one of the jpg sketches she send me earlier.

**Comment by KizzyCode** [comment]
Happens to me too; every damn time^^
Would it be ok if I created an alpha-PNG (white-BG → alpha and so on)? Because the image is pretty nice even if it wouldn't become the new mascot...

**Comment by kamisori** [comment]
go ahead

**Comment by kamisori** [comment]
Actually, my wife took on the effort to make it transparent herself

![hugs](https://user-images.githubusercontent.com/293332/36702392-353fa0f2-1b57-11e8-9313-cc58129d5ef2.png)

**Comment by Ybalrid** [comment]
This is great!

**Comment by KizzyCode** [comment]
Nice; this is great 👍🏻
Kudos to your wife 👏🏻
And of course: feel hugged 🤗

**Comment by Serkan-devel** [comment]
@kamisori /)

**Comment by Member1221** [comment]
![freehugsbsdwallp](https://user-images.githubusercontent.com/7032834/39411088-4a785c70-4c03-11e8-83b9-e39fa323d5d3.png)
I turned it in to a wallpaper, cus why not?
